Mathlib/MeasureTheory/Decomposition/SignedLebesgue.lean

```diff
@@ -315,13 +309,13 @@ private theorem eq_singularPart' (t : SignedMeasure α) {f : α → ℝ} (hf : M
     JordanDecomposition.toSignedMeasure]
   congr
   -- NB: `measurability` proves this `have`, but is slow.
-  -- TODO(#13864): reinstate faster automation, e.g. by making `fun_prop` work here
+  -- XXX: fun_prop doesn't work yet...
```

Comment:
`XXX` is pretty useless as an indicator. Can we just leave `TODO`?

Also, flagging this for myself to understand in depth why this doesn't work.

Reply:
Sure - obviously, my preferred outcome is to make `fun_prop` work. If fixing that is for another PR, feel free to apply the following:
```suggestion
  -- TODO: make `fun_prop` able to handle this